### ISSUE TITLE
The website is blank

### ISSUE THREAD

**Issue by ventedruby** [JimMarshall35/JimMarshall35.github.io#1]
The website was blanked a week ago. Was this intentional?

**Comment by RepeatGitHub** [comment]
I think it may have been, one of the commits says "made blank" for some reason. I wonder why Jim did this.

**Comment by ventedruby** [comment]
kinda bummed. I was using it to make a rom hack.

**Comment by ventedruby** [comment]
Wondering if the website will come back. Does anyone know of any good alternatives similar to this one?

**Comment by JimMarshall35** [comment]
Oh I'm sorry - I thought this was an automated github email - I'll put it
back on when I get chance

**Comment by JimMarshall35** [comment]
I was going to use the free github hosting for something else - but I'll keep this in as a link somwhere on the page - I had no idea anyone used it or knew about it

**Comment by JimMarshall35** [comment]
this is the same thing in a different repo if you want to try and get it working yourself in the mean time https://github.com/JimMarshall35/nes-sprite-editor

**Comment by JimMarshall35** [comment]
I've reverted the commit - you can use it again - enjoy! Let me know any improvements you might think of and I might revisit this and add them one day if there's a unfulfilled need for a web app like this

**Comment by ventedruby** [comment]
This tool was the first sprite editor I’ve found online that had a GOOD interface. Even though yours is barebones, it is 10x better than another online NES sprite tool I’ve found. Being able to arrange the tiles on a grid is very convenient. I struggled with visualizing an entire sprite without the other missing tiles. 

As an artist, I like using these kinds of tools, but none are compatible with new Mac versions (hell, none are even made for Mac at all!) You wouldn’t believe how useful this tool is. 

Being able to work on my ROMhack at school (in my free time, by the way.) is SO convenient and saves so much time. 

I have one issue, but it’s probably just an issue with old browsers, but the 4 color palette squares appear black on browsers like Safari. They will change to other colors you set, however. But like I said, it’s probably just an old browser thing.

There’s a bug where if you cross the edges of tiles on the tile arranger, you will paint the right side of the tile you were crossing over from. 

Anyways, thanks a lot. 

This message was sent via DOS command prompt.

**Comment by JimMarshall35** [comment]
Thanks for the feedback - glad you like it! I'd like to pick this back up and fix the issues you mentioned, and also add more features, but I might not. Some I intended to do were undo/redo, ability to resize the tile arranger, ability to manually select the area of the rom where the tiles are for games like legend of zelda that keep them in different places, ability to write a blank binary file with tiles in it for developers of new games, ability to zoom in and out when editing tiles. This is written in typescript and uses angular - I'm a bit rusty on these nowerdays - the reason I deleted it was because I wanted to experiment with C++/ OpenGL  compiled to web assembly - If I manage get that setup working I might alternatively write a new and better NES editor using those technologies, which I'm now more familiar with

**Comment by ventedruby** [comment]
I really appreciate your response. Have you thought about hosting it through a different GitHub account? I wouldn’t want it to keep you from working on those new projects.

 I had mirrored the site on my GitHub page, with credit to you. If you want me to take it down, I can.

By the way, the hack I’m working on wouldn’t have even started development without your sprite editor. I wish I was kidding! Haha.

I’m excited to see what you will create!

Thanks again. - BDAYHax/banditruby

**Comment by JimMarshall35** [comment]
you needn't worry about that - I can just host it through a local server on my computer to develop :) I don't really need to publicly host anything until it's complete - I just wanted to for some reason haha. 

I've got a lot of projects on the go at the moment - too many really - and I'm ALWAYS tempted to start more! 

I love retro games though - some of the most fun I've ever had coding was through writing my gameboy snake game and I particularly enjoyed doing the graphics for it. I'd love to write a more ambitious Gameboy game one day and would really be interested in collaborating with others on it if you're interested at all?

 I think it would be really fun to create some gameboy tools to support it as well, although there's some excellent ones out there already for windows at least. NES is all very well and good, but as a brit from the UK who was born in 1995 it doesn't hold that much nostalgia for me - the original gameboy and gb colour on the other hand I used to play all the time!
